src/components/Link/tests/Link.test.tsx

```diff
@@ -81,4 +83,42 @@ describe('<Link />', () => {
       expect(link.find(Icon).exists()).toBe(false);
     });
   });
+
+  describe('monochrome link', () => {
+    it('outputs a monochrome unstyled link if rendered within a banner', () => {
+      const link = mountWithAppProvider(
+        <WithinBannerContext.Provider value>
```

Comment:
Should we use `Banner` rather than `WIthinBannerContext`?

Reply:
Would that create naming conflicts when it's used in the `Banner` module?

Reply:
I'm not sure I follow, would you mind expanding?